OracleFMWInfrastructure/README.md

```diff
@@ -0,0 +1,135 @@
+
+The certification of Oracle FMW Infrastructure on Docker does not require the use of any file presented in this repository. Customers and users are welcome to use them as starters, and customize/tweak, or create from scratch new scripts and Dockerfiles.
+
+
+## How to build and run
+This project offers a sample Dockerfile and scripts to build a Oracle Fusion Middleware Infrastructue 12cR2 (12.2.1.2) image. To assist in building the image, you can use the [buildDockerImage.sh](dockerfiles/buildDockerImage.sh) script. See below for instructions and usage.
+
+The `buildDockerImage.sh` script is just a utility shell script that takes the version of the image that needs to be built. Expert users are welcome to directly call `docker build` with their prefered set of parameters.
+
+
+### Building Oracle JDK (Server JRE) base image
+**IMPORTANT:**You must first download the Oracle Server JRE binary and drop in folder `../OracleJava/java-8` and build that image. For more information, visit the [OracleJava
+](../OracleJava) folder's [README](../OracleJava/README.md) file.
+
+        $ cd ../OracleJava/java-8
+        $ sh build.sh
+
+### Building Oracle FMW Infrastructure Docker Install Images
+**IMPORTANT:** you have to download the binary of Oracle FMW Infrastructure and put it in place (see `.download` files inside dockerfiles/<version>).
+
+Before you build, either pull the corresponding version of the Oracle Server JRE 8 image from the Oracle Container Regisrty, or manually create the Java image using the Dockerfile and scripts in GitHub as described above.  
+
+Before you build, choose which version of the image you want to build,then download the required packages (see .download files) and drop them in the folder of your distribution version of choice. Then go into the **dockerfiles** folder and run the **buildDockerImage.sh** script as root.
+
+        $ sh buildDockerImage.sh
+        Usage: buildDockerImage.sh -v [version] [-s]
+        Builds a Docker Image for Oracle FMW Infrastructure.
+
+        Parameters:
+           -v: version to build. Required.
+           Choose : 12.2.1.2
+           -c: enables Docker image layer cache during build
+           -s: skips the MD5 check of packages
+
+        LICENSE CDDL 1.0 + GPL 2.0
+
+        Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
+
+**IMPORTANT:** the resulting images will have a domain with an Admin Server and one Managed Server by default. You must extend the image with your own Dockerfile, and create your domain using WLST.
+
+
+### Sample FMW Infrastructure Domain 
+The image **oracle/fmw-infrastructure:12.2.1.2** will configure a **base_domain** with the following settings:
+
+ * Admin Username: `weblogic`
+ * Admin Password: `Auto generated` 
+ * Oracle Linux Username: `oracle`
+ * Oracle Linux Password: `welcome1`
```

Comment:
Though, I can't actually see anywhere that this is set, so I'm not sure this is valid.